app/assets/src/components/views/amr_heatmap/AMRHeatmapVis.jsx

```diff
@@ -193,11 +104,28 @@ export default class AMRHeatmapVis extends React.Component {
       }
     }
 
-    let values = METRICS.map(metric => {
-      const value = amrCountForNode ? amrCountForNode[metric.key] : 0;
+    let values = metrics.map(metric => {
```

Comment:
So we show `N/A` if and only if the sample is too old to have the evaluated metrics, correct? Can you add a comment explicitly explaining that?

Reply:
Added.